docs/getting-started.md

```diff
@@ -122,7 +134,13 @@ repository:
     url: "https://github.com/jshttp/mime-types.git"
     revision: "7c4ce23d7354fbf64c69d7b7be8413c4ba2add78"
     path: ""
-  config: {}
+  # Will be added, in case .ort.yml file is provided. Otherwise empty object
```

Comment:
Nit: "Will only be present if an '.ort.yml' configuration file with scope excludes was provided. Otherwise this is an empty object."